docs/source/conf.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+
+import idelib
+
+# -- Project information -----------------------------------------------------
+
+project = u'idelib'
+copyright = u'2020, Mide Technology Corp.'
```

Comment:
If this is shown anywhere, it should probably be updated before release.

Reply:
by "updated" do you mean change it to 2021?
(I did ^that in f34863e2684b9ee9eddd485954b5e896cdc7e481)

Reply:
Yeah, that's what I meant. I think the Lab and other things still reference Mide (as opposed to referencing enDAQ or Hutchinson)